osbs/api.py

```diff
@@ -713,6 +713,47 @@ def get_build_logs(self, build_id, follow=False, build_json=None, wait_if_missin
 
         return logs
 
+    def _parse_build_log_entry(self, entry):
```

Comment:
Very minor: since this method doesn't use `self`, it could be made into a "static method", i.e. a function defined in the class but without adding the instance as the first argument:
```
@staticmethod
def _parse_build_log_entry(entry):
```
This is what landscape.io is on about here: https://landscape.io/github/projectatomic/osbs-client/pulls/665